scripts/pages/search.py

```diff
@@ -79,9 +81,17 @@
         while bracketLevel <= targetLevel:
             initSearch.append("Tiqit.search.bracketRightShift(document.getElementById('openBracket%d.%d').childNodes[1]);" % (i, bracketLevel))
             bracketLevel += 1
-        
+
+    i += 1
+
+i = 1
+while args.has_key('field%d' % i):
+    initSearch.append("Tiqit.search.setRowValues(%d, '%s', '%s', '%s', '%s');" \
+          % (i, args['field%d' % i], args['rel%d' % i], args['val%d' % i],
+             args['operation%d' % i]))
     i += 1
 
+
```

Comment:
Remove one of the new lines of whitespace? Don't think we need/want to have two.